src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,37 @@
+# How to run tests on the Raspberry Pi
+
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
```

Comment:
I believe only runtime is required if you're not building on RPi directly

Reply:
Probably, but I'm showing how to build on the Pi a few lines down. And those who really build this project on their own will want the SDK installed.